tsfile/src/main/java/org/apache/iotdb/tsfile/read/common/Chunk.java

```diff
@@ -60,12 +67,51 @@ public void setDeleteIntervalList(List<TimeRange> list) {
     this.deleteIntervalList = list;
   }
 
-  public void mergeChunk(Chunk chunk) {
-    chunkHeader.mergeChunkHeader(chunk.chunkHeader);
-    ByteBuffer newChunkData = ByteBuffer
-        .allocate(chunkData.array().length + chunk.chunkData.array().length);
-    newChunkData.put(chunkData.array());
-    newChunkData.put(chunk.chunkData.array());
+  public void mergeChunk(Chunk chunk) throws IOException {
```

Comment:
this method needs a lot of comments. Will review it after supplying comments.

Reply:
already add, you can start reviewing